3p/iframe-messaging-client.js

```diff
@@ -59,7 +59,7 @@ export class IframeMessagingClient {
   /**
    *  Send a postMessage to Host Window
    *  @param {object} message The message to send.
-   *  @private
+   *  @protected
```

Comment:
I switched to protected because its child classes need to be able to access it, but nothing outside the class or its child class should be accessing it. Leaving the underscores was an oversight